package.json

```diff
@@ -3,15 +3,34 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
```

Comment:
You're adding a lot of `devDependencies` into `dependencies`. Would you mind transferring them to `devDependencies`?

Reply:
Agree (it's a default behavior of eject, forgot to move it to devDeps)